### PR TITLE
Explicitly specify metadata filters for collections through queryable…

### PR DESCRIPTION
**Pull request by m-mohr**
Implements #396

### CODE REVIEW

**Review by PondiB — APPROVED** [review]
For familiarization on my end.

File: openapi.yaml [hunk 1]
```diff
@@ -1436,6 +1431,9 @@ paths:
                   - rel: license
                     href: https://scihub.copernicus.eu/twiki/pub/SciHubWebPortal/TermsConditions/Sentinel_Data_Terms_and_Conditions.pdf
                     type: application/pdf
+                  - rel: http://www.opengis.net/def/rel/ogc/1.0/queryables
+                    href: https://example.openeo.org/api/collections/Sentinel-2A/queryables
```
Comment (soxofaan) [inline openapi.yaml:1435]:
```suggestion
                    href: https://openeo.example/api/collections/Sentinel-2A/queryables
```

(.example is a dedicated tld for examples, avoids risk of having a URL here that might start working in the future)

File: openapi.yaml [hunk 2]
```diff
@@ -1574,6 +1572,67 @@ paths:
...
+            A JSON Schema defining the queryables.
+
+            It is RECOMMENDED to dereference all "$refs".
+          content:
+            application/schema+json:
+              schema:
+                $ref: '#/components/schemas/json_schema'
+              example:
+                $schema: https://json-schema.org/draft/2019-09/schema
+                $id: https://example.com/api/v1.0/collections/Sentinel-2A/queryables
```
Comment (soxofaan) [inline openapi.yaml:1614]:
```suggestion
                $id: https://openeo.example/api/v1.2/collections/Sentinel-2A/queryables
```

**Review by soxofaan — APPROVED** [review]
just minor notes, and going to approve already

File: openapi.yaml (hunk 1 above)
Comment (m-mohr) [reply openapi.yaml:1435]: Fine, but I think it should be a separate issue that changes it for all examples. :-)

Comment (soxofaan) [reply openapi.yaml:1435]: sure, feel free to merge this PR already

Comment (soxofaan) [reply openapi.yaml:1435]: spin of: #489

**Review by clausmichele — APPROVED** [review]

**Pull request merged**